WSL/release-notes.md

```diff
@@ -10,6 +10,14 @@ ms.localizationpriority: high
 
 # Release Notes for Windows Subsystem for Linux
 
+## Build 21286
+For general Windows information on build 21286 visit the [Windows blog](https://blogs.windows.com/windows-insider/2021/01/06/announcing-windows-10-insider-preview-build-21286/).
+
+* Added a boot.command option to /etc/wsl.conf to enable start up commands.
```

Comment:
`boot.command` syntax may not be familiar to all users. It may be written in ini style. e.g.

```ini
[boot]
command="echo hello"
```

Reply:
Agreed, I will take a look to see if we can add it to the docs under 'preview' somewhere so users can discover this feature.